src/client/components/dashboard/Card.tsx

```diff
@@ -0,0 +1,37 @@
+import avatarLogo from "@assets/logos/avatar_placeholder.png";
+import { IUserPublic } from "@typing/user.interface";
+import { FunctionComponent } from "react";
+import Card from "react-bootstrap/Card";
+import styles from "../../styles/dashboard/dashboard.module.scss";
+
+interface Props {
+  user: IUserPublic;
+  nbrPost: number;
+}
+
+const DashBoardCard: FunctionComponent<Props> = ({ user, nbrPost }: Props) => {
+  return (
+    <div>
+      <h3> Mon Profil</h3>
+      <Card className={styles.card}>
+        {/* Next considère le logo comme un objet possédant un attribut src */}
+        <Card.Img
+          variant="top"
+          src={avatarLogo.src}
+          className={styles.cardImage}
+        />
+        <Card.Body>
+          <span className={styles.badge}>veteran</span>
+          <Card.Text className={styles.cardText}>
+            <p>Nom</p>
+            <span>{user.firstName ? user.firstName : "Votre nom"}</span>
```

Comment:
```suggestion
            <span>{getUsername(user) ?? "Votre nom"}</span>
```
Vous l'avez sans doute pas vu mais il y a une fonction pour formatter le nom de l'utilisateur

Reply:
hooooo

Reply:
Tu as oublié ça ? 😂

Reply:
C'est bon mdr